File: python/print_params_env.py
# Put the main logic of the task in the main function.
import os

def main(params):
    print(params)

    return [
        {
            "environment": os.environ.get("AIRPLANE_ENV_SLUG"),
            "airplane_resources_version": os.environ.get("AIRPLANE_RESOURCES_VERSION"),
            "airplane_resources": os.environ.get("AIRPLANE_RESOURCES"),
        },
    ]

File: node/print_params_env.ts
export default async function(params) {
  console.log(params);

  return [
    {
      "environment": process.env.AIRPLANE_ENV_SLUG,
      "airplane_resources_version": process.env.AIRPLANE_RESOURCES_VERSION,
      "airplane_resources": process.env.AIRPLANE_RESOURCES
    },
  ];
}
